### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by GuillaumeCisco**

### CODE REVIEW

**Comment by Kelvin-M** [comment]
Nice work, I need to test it !

File: backend/requirements.txt [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
+pycryptodomex>=3.9.4
+pyOpenSSL==19.1.0
+raven==6.10.0
+requests==2.22.0
+sentry-sdk==0.14.1
+six>=1.14.0
+tldextract==2.2.2
+uwsgi==2.0.18
 zxcvbn==4.4.28
 git+git://github.com/hyperledger/fabric-sdk-py.git@36cc15021f74c11c7ae3196e380a5275c220145f # fabric-sdk-py==0.8.1
```
Comment (inalgnu) [inline backend/requirements.txt]: Since fabric-sdk-py 0.8.1 was released, we could use a release tag here?

```
fabric-sdk-py==0.8.1

Comment (Kelvin-M) [reply backend/requirements.txt]: I remembered that there was an issue on the 0.8.1, we should test it :)

Comment (Kelvin-M) [reply backend/requirements.txt]: I've checked, the commit '36cc15...' is later than 0.8.1
I'm currently testing the last commit of fabric-sdk-py

**Comment by Kelvin-M** [comment]
@GuillaumeCisco  I've updated the fabric sdk py (tested on docker and skaffold)

**Comment by Kelvin-M** [comment]
@GuillaumeCisco Good for me, feel free to merge if you do not have more dependencies to update. See my update on fabric version too :)

**Comment by GuillaumeCisco** [comment]
@inalgnu is this update ok for you?
If yes, do not hesitate to merge it.

**Comment by inalgnu** [comment]
I'm still wondering why we are pointing a commit instead of using a tag for fabric-sdk-py (If I remember well we did that because the feature was not released yet). It seems not to be the case anymore? do you confirm @GuillaumeCisco?

**Comment by Kelvin-M** [comment]
> I'm still wondering why we are pointing a commit instead of using a tag for fabric-sdk-py (If I remember well we did that because the feature was not released yet). It seems not to be the case anymore? do you confirm @GuillaumeCisco?

Because, fabric-sdk-py 0.8.1 is broken, so we need a commit later than the tag because @GuillaumeCisco  have fixed an issue introduced in this release. As there is no new release since we have to use a commit number for the version !

**Comment by GuillaumeCisco** [comment]
Baohua still did not give his agreement for releasing a 8.2 version.
That's why we use last commit on master.

**Comment by inalgnu** [comment]
That makes sense. Thank's

**Pull request merged**